components/net/cookie.rs

```diff
@@ -62,11 +62,12 @@ impl Cookie {
         let (persistent, expiry_time) = match (cookie.max_age(), cookie.expires()) {
             (Some(max_age), _) => (
                 true,
-                Some(at(
-                    now().to_timespec() + Duration::seconds(max_age.num_seconds())
-                )),
+                Some(SystemTime::now() + Duration::from_nanos(max_age.num_seconds() as u64)),
+            ),
+            (_, Some(expires)) => (
+                true,
+                Some(SystemTime::UNIX_EPOCH + Duration::from_nanos(expires.tm_sec as u64)),
```

Comment:
Should this use `Duration::from_secs` in both of these cases?

Reply:
I think this why I unit test did not pass, I will change it and check